src/main/java/frc/robot/subsystems/Climber.java

```diff
@@ -68,15 +87,15 @@ public void runClimber() {
     }
 
     public void raiseRobot() {
-        climber.set(TalonFXControlMode.PercentOutput, 1.0);
+        climber.set(TalonFXControlMode.Velocity, climberRpms);
```

Comment:
Why don't we want to climb at max speed?

Reply:
because max speed is too fast (that is what the climber team told us)

Reply:
Ok